tests/templates/test_embeddings/test_basis.py

```diff
@@ -239,33 +239,24 @@ def test_jax(self, tol):
         res = circuit(jnp.array(2))
         assert qml.math.allclose(res, res2, atol=tol, rtol=0)
 
-    @pytest.mark.parametrize(
-        "device_name",
-        ["default.qubit", "lightning.qubit"],
-    )
     @pytest.mark.jax
-    def test_jax_jit(self, tol, device_name):
-        """Tests the jax-jit interface."""
+    def test_jax_jit(self, tol):
+        """Tests compilation with JAX JIT."""
 
         import jax
         import jax.numpy as jnp
 
         features = jnp.array([0, 1, 0])
```

Comment:
BasisState also accepts integers instead of binary lists. Could you check that it is JIT compatible in that case too?